src/options/view.rs

```diff
@@ -302,12 +302,25 @@ impl SizeFormat {
     /// strings of digits in your head. Changing the format to anything else
     /// involves the `--binary` or `--bytes` flags, and these conflict with
     /// each other.
-    fn deduce(matches: &MatchedFlags<'_>) -> Result<Self, OptionsError> {
+    fn deduce<V: Vars>(matches: &MatchedFlags<'_>, vars: &V) -> Result<Self, OptionsError> {
+        let env_var = match vars.get(vars::EZA_SIZE_STYLE) {
+            Some(s) => match s.to_string_lossy().to_string().as_ref() {
+                "binary" => Some("binary"),
+                "bytes" => Some("bytes"),
+                "decimal" => Some("decimal"),
+                _ => None,
+            },
+            _ => None,
+        };
+
         let flag = matches.has_where(|f| f.matches(&flags::BINARY) || f.matches(&flags::BYTES))?;
 
-        Ok(match flag {
-            Some(f) if f.matches(&flags::BINARY) => Self::BinaryBytes,
-            Some(f) if f.matches(&flags::BYTES) => Self::JustBytes,
+        Ok(match (env_var, flag) {
```

Comment:
Command line should take priority over the environment variable.

Reply:
I agree with that BUT, at that point this PR become kinda unuseful. So I might as well close it ?

Reply:
up @cfxegbert